lib/modules/manager/droneci/extract.ts

```diff
@@ -3,26 +3,64 @@ import { newlineRegex, regEx } from '../../../util/regex';
 import { getDep } from '../dockerfile/extract';
 import type { PackageDependency, PackageFile } from '../types';
 
+export function getDroneDep(
+  currentFrom: string | null | undefined,
+  replaceString: string | undefined
+): PackageDependency {
+  const dep = getDep(currentFrom);
+  logger.debug(
```

Comment:
```suggestion
  logger.trace(
```

Reply:
this one was debuf from before, I just moved the code in a helper method. Anyways, changed to trace